idx/cassandra/cassandra.go

```diff
@@ -538,19 +534,17 @@ func (c *CasIdx) deleteDefAsync(key schema.MKey, part int32) {
 	}()
 }
 
-func (c *CasIdx) Prune(oldest time.Time) ([]idx.Archive, error) {
-	pre := time.Now()
-	pruned, err := c.MemoryIdx.Prune(oldest)
-	statPruneDuration.Value(time.Since(pre))
+func (c *CasIdx) Prune(now time.Time) ([]idx.Archive, error) {
+	pruned, err := c.MemoryIdx.Prune(now)
+	statPruneDuration.Value(time.Since(now))
 	return pruned, err
 }
 
 func (c *CasIdx) prune() {
 	ticker := time.NewTicker(pruneInterval)
-	for range ticker.C {
-		log.Debug("cassandra-idx: pruning items from index that have not been seen for %s", maxStale.String())
-		staleTs := time.Now().Add(maxStale * -1)
-		_, err := c.Prune(staleTs)
+	for now := range ticker.C {
+		log.Debug("cassandra-idx: pruning items")
```

Comment:
I think we should have an info level log message for when a prune starts, and an info message when the prune completes.  The message on completion should log how long the prune took and number of items pruned.

Reply:
we already have such info logs for the memory index and the cassandra index doesn't really do any work beyond that, but I think it's still useful. in fact i'm making the logs from both more consistent so that it'll look obvious in the logs, both memory and cassandra index logging their own steps.